src/frontend/pages/login/Login.js

```diff
@@ -0,0 +1,84 @@
+import { Link } from "react-router-dom";
+import { useAuth } from "../../contexts/auth-context";
+import { useLocation, useNavigate } from "react-router-dom";
+import axios from "axios";
+import "./login.css";
+
+export function Login() {
+  const { authState, authDispatch } = useAuth();
+  const { user, error } = authState;
+  const navigate = useNavigate();
+  const location = useLocation();
+
+  const loginHandler = () => {
+    try {
+      const response = axios.post(`/api/auth/login`, {
+        email: user.email,
+        password: user.password,
+      });
+      localStorage.setItem("token", response.data.encodedToken);
+      authDispatch({ type: "ERROR", payload: response.data.encodedToken });
+      navigate("navigate(location.state.from.pathname, { replace: true })");
```

Comment:
can you please check your syntax here? 
It should be 
**navigate(`${location?.state?.from?.pathname}`, { replace: true } )**